glustercli/cmd/volume-reset.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	volumeResetCmdHelpShort = "Reset volume options"
+	volumeResetCmdHelpLong  = "Reset options on a specified gluster volume. Needs a volume name and at least one option"
+)
+
+var (
+	flagForce, flagResetAll bool
+
+	volumeResetCmd = &cobra.Command{
+		Use:   "reset <volname> <option>",
+		Short: volumeResetCmdHelpShort,
+		Long:  volumeResetCmdHelpLong,
+		Args:  volumeResetCmdArgs,
+		Run:   volumeResetCmdRun,
+	}
+)
+
+func init() {
+	volumeResetCmd.Flags().BoolVar(&flagForce, "force", false, "Force reset the volume option")
+	volumeResetCmd.Flags().BoolVar(&flagResetAll, "all", false, "Reset all the volume options")
+	volumeCmd.AddCommand(volumeResetCmd)
+}
+
+func volumeResetCmdArgs(cmd *cobra.Command, args []string) error {
```

Comment:
you can remove this function implementation itself, 

you can do something like this

```
var volumeResetCmd = &cobra.Command{
	Use:   " reset <volname> <option>",
	Short: volumeResetCmdHelpShort,
        Long:  volumeResetCmdHelpLong,
	Args:  cobra.RangeArgs(1, 2),
	Run: func(cmd *cobra.Command, args []string) {
		volname := cmd.Flags().Args()[0]
		//next operation
	},
}
```